staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -44,56 +44,58 @@ const (
 type KubeSchedulerConfiguration struct {
 	metav1.TypeMeta `json:",inline"`
 
-	// Parallelism defines the amount of parallelism in algorithms for scheduling a Pods. Must be greater than 0. Defaults to 16
+	// The amount of parallelism in algorithms for scheduling Pods.
+	// Must be greater than 0. Defaults to 16.
 	Parallelism *int32 `json:"parallelism,omitempty"`
 
-	// LeaderElection defines the configuration of leader election client.
+	// The configuration of leader election client.
```

Comment:
is the new docs style now to not start with the field name?

Reply:
The intent of this change is to avoid exposing Go field name to users who are editing JSON or YAML. The Go field names all have their first letter capitalized so that they are "public". However, in JSON or YAML (the user facing config files), these field names are spelled in a different way.